Nodejs-GitHub-Sponsors.md

```diff
@@ -0,0 +1,34 @@
+# Node.js GitHub sponsors
```

Comment:
I think we should note somewhere that the funds received via GitHub Sponsors will be fiscally hosted by the OpenJS Foundation.

Reply:
I think that’s already the `Account details` section